### ISSUE TITLE
Create documentation for using HOPR-chat in different OS

### ISSUE THREAD

**Issue by 0xjjpa** [hoprnet/hopr-core#69]
**Written**
- [ ] Create written documentation for running HOPR in your machine
  - [ ] *Add instructions on how to run HOPR-chat locally using `node`
    - [ ] *Add instructions on how to install `node` in your OS
    - [ ] *Add instructions on how to install HOPR-chat using `node`
  - [x] Add instructions on how to run HOPR-chat locally using `Docker`
    - [x] Add instructions on how to install `Docker` in your OS
    - [x] Add instructions on how to pull HOPR-chat `Docker` image
  - [x] Add instructions on how to use HOPR-chat
    - [x] How to get my address
    - [x] How to send a message to another node
- [x] Add documentation as separate Markdown document
- [x] Publish documentation within GitBook

_* Less priority, we'll focus on `Docker` usage for now._

**Visual**
- [x] Create video for installing docker inside a machine
  - [x] Create one for windows
- [x] Create a video for running HOPR w/Docker in a machine
  - [x] Create one for windows
  - [x] Create one for OSx
  - [x] Create one for ubuntu

**Comment by 0xjjpa** [comment]
Added `macos` documentation on https://docs.hoprnet.io/home/getting-started/quickstart/macos-quickstart, which we can use for the time being for Linux as well. Thus, I've marked all the tasks for `ubuntu` as completed. We’ll reopen this one if seems its needed.

I've integrated a series of gifs to replace the videos for now. We'll split the instructions for documenting the node setup in a separate task.
